app/helpers/repository_helper.rb

```diff
@@ -0,0 +1,28 @@
+# Returns information about the repository that is configured in the
+# configuration file
+module RepositoryHelper
+  # Returns configuration properties for the current configured
+  # repository
+  def repository_config
+    conf = {}
+    conf["IS_REPOSITORY_ADMIN"] =
+        MarkusConfigurator.markus_config_repository_admin?
+    conf["REPOSITORY_PERMISSION_FILE"] =
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.